puppet-lint-trailing_comma-check.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
     A puppet-lint plugin to check for missing trailing commas.
   EOF
 
-  spec.required_ruby_version = Gem::Requirement.new(">= 2.0".freeze)
+  spec.required_ruby_version = Gem::Requirement.new(">= 2.5".freeze)
```

Comment:
@ekohl when we start dropping ruby 2.x, should we directly bump to ruby 2.6 ir 2.7? 2.5 is EoL zpstream already, but puppet 6 uses it (which is also almost EoL).

Reply:
I think with lint we can safely make it 2.7+ given that you can run lint just once. That means you can keep supporting older Puppet 6 versions.